downstream/titles/hub/managing-content/master.adoc

```diff
@@ -10,7 +10,25 @@ include::attributes/attributes.adoc[]
 include::{Boilerplate}[]
 
 include::hub/assembly-managing-cert-valid-content.adoc[leveloffset=+1]
+include::hub/assembly-syncing-to-cloud-repo.adoc[leveloffset=+1]
```

Comment:
Do you want to keep the original ToC hierarchy after flattening the nested assemblies?
If you do, the "flattened" assemblies should have leveloffset=2